### ISSUE TITLE
Check failed: *base_computed_style_ == *computed_style

### ISSUE THREAD

**Issue by ckerr** [electron/electron#11949]
Electron version is master on Sun Feb 18, commit 55d4d74624d6546947547c7bd2105907cdb1db93.

Platform is Ubuntu 17.10

Steps to reproduce:
 1. Make debug build of electron
 2. Run electron-api-demos: `./out/D/electron ../electron-api-demos/`
 3. Click on "About" in the lower left of the Demo window

Results:

```text
[1017:0218/165129.021274:FATAL:ElementAnimations.cpp(112)] Check failed: *base_computed_style_ == *computed_style. 
#0 0x7f7b3a9bfdbb base::debug::StackTrace::StackTrace()
#1 0x7f7b3a9be84c base::debug::StackTrace::StackTrace()
#2 0x7f7b3aa31d23 logging::LogMessage::~LogMessage()
#3 0x7f7b45b6852f <unknown>
#4 0x7f7b45e859cc <unknown>
#5 0x7f7b45e87089 blink::StyleResolver::PseudoStyleForElementInternal()
#6 0x7f7b45e8736a blink::StyleResolver::PseudoStyleForElement()
#7 0x7f7b45f29484 blink::Element::GetUncachedPseudoStyle()
#8 0x7f7b45f28cb7 blink::Element::PseudoStyle()
#9 0x7f7b46024596 blink::PseudoElement::CustomStyleForLayoutObject()
#10 0x7f7b45f21d99 blink::Element::StyleForLayoutObject()
#11 0x7f7b45f225b7 blink::Element::RecalcOwnStyle()
#12 0x7f7b45ee733b blink::Element::RecalcStyle()
#13 0x7f7b45f22e0c blink::Element::UpdatePseudoElement()
#14 0x7f7b45ee7414 blink::Element::RecalcStyle()
#15 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#16 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#17 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#18 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#19 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#20 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#21 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#22 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#23 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#24 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#25 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#26 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#27 0x7f7b45efd957 blink::Document::UpdateStyle()
#28 0x7f7b45ef959c blink::Document::UpdateStyleAndLayoutTree()
#29 0x7f7b463685f1 blink::LocalFrameView::UpdateStyleAndLayoutIfNeededRecursiveInternal()
#30 0x7f7b463668ce blink::LocalFrameView::UpdateStyleAndLayoutIfNeededRecursive()
#31 0x7f7b4636554a blink::LocalFrameView::UpdateLifecyclePhasesInternal()
#32 0x7f7b46365232 blink::LocalFrameView::UpdateAllLifecyclePhases()
#33 0x7f7b46ca1e4b blink::PageAnimator::UpdateAllLifecyclePhases()
#34 0x7f7b46ca4175 blink::PageWidgetDelegate::UpdateAllLifecyclePhases()
#35 0x7f7b42389cb7 blink::WebViewImpl::UpdateAllLifecyclePhases()
#36 0x7f7b46399b52 blink::WebViewFrameWidget::UpdateAllLifecyclePhases()
#37 0x7f7b3dec498b content::RenderWidget::UpdateVisualState()
#38 0x7f7b3dcc866a content::RenderWidgetCompositor::UpdateLayerTreeHost()
#39 0x7f7b49ad70ed cc::LayerTreeHost::RequestMainFrameUpdate()
#40 0x7f7b49bc79ac cc::ProxyMain::BeginMainFrame()
#41 0x7f7b49bc48b4 <unknown>
#42 0x7f7b49bc475f <unknown>
#43 0x7f7b49bc46c8 <unknown>
#44 0x7f7b49bc45de <unknown>
#45 0x7f7b3a97b09e <unknown>
#46 0x7f7b3a9c5a8e base::debug::TaskAnnotator::RunTask()
#47 0x7f7b43361674 blink::scheduler::TaskQueueManager::ProcessTaskFromWorkQueue()
#48 0x7f7b4335e766 blink::scheduler::TaskQueueManager::DoWork()
#49 0x7f7b43369b9c <unknown>
#50 0x7f7b43369acf <unknown>
#51 0x7f7b43369a43 <unknown>
#52 0x7f7b4336995c <unknown>
#53 0x7f7b3a97b09e <unknown>
#54 0x7f7b3a9c5a8e base::debug::TaskAnnotator::RunTask()
#55 0x7f7b3aa5b6cd base::MessageLoop::RunTask()
#56 0x7f7b3aa5b957 base::MessageLoop::DeferOrRunPendingTask()
#57 0x7f7b3aa5bc6f base::MessageLoop::DoWork()
#58 0x7f7b3aa6e358 base::MessagePumpDefault::Run()
#59 0x7f7b3aa5b098 base::MessageLoop::Run()
#60 0x7f7b3ab060ed base::RunLoop::Run()
#61 0x7f7b3defcc24 <unknown>

Received signal 6
#0 0x7f7b3a9bfdbb base::debug::StackTrace::StackTrace()
#1 0x7f7b3a9be84c base::debug::StackTrace::StackTrace()
#2 0x7f7b3a9bf7c5 <unknown>
#3 0x7f7b34e93150 <unknown>
#4 0x7f7b2ea880bb gsignal
#5 0x7f7b2ea89f5d abort
#6 0x7f7b3a9bc846 <unknown>
#7 0x7f7b3a9bc828 base::debug::BreakDebugger()
#8 0x7f7b3aa321a4 logging::LogMessage::~LogMessage()
#9 0x7f7b45b6852f <unknown>
#10 0x7f7b45e859cc <unknown>
#11 0x7f7b45e87089 blink::StyleResolver::PseudoStyleForElementInternal()
#12 0x7f7b45e8736a blink::StyleResolver::PseudoStyleForElement()
#13 0x7f7b45f29484 blink::Element::GetUncachedPseudoStyle()
#14 0x7f7b45f28cb7 blink::Element::PseudoStyle()
#15 0x7f7b46024596 blink::PseudoElement::CustomStyleForLayoutObject()
#16 0x7f7b45f21d99 blink::Element::StyleForLayoutObject()
#17 0x7f7b45f225b7 blink::Element::RecalcOwnStyle()
#18 0x7f7b45ee733b blink::Element::RecalcStyle()
#19 0x7f7b45f22e0c blink::Element::UpdatePseudoElement()
#20 0x7f7b45ee7414 blink::Element::RecalcStyle()
#21 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#22 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#23 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#24 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#25 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#26 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#27 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#28 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#29 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#30 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#31 0x7f7b45ee6e45 blink::ContainerNode::RecalcDescendantStyles()
#32 0x7f7b45ee74c4 blink::Element::RecalcStyle()
#33 0x7f7b45efd957 blink::Document::UpdateStyle()
#34 0x7f7b45ef959c blink::Document::UpdateStyleAndLayoutTree()
#35 0x7f7b463685f1 blink::LocalFrameView::UpdateStyleAndLayoutIfNeededRecursiveInternal()
#36 0x7f7b463668ce blink::LocalFrameView::UpdateStyleAndLayoutIfNeededRecursive()
#37 0x7f7b4636554a blink::LocalFrameView::UpdateLifecyclePhasesInternal()
#38 0x7f7b46365232 blink::LocalFrameView::UpdateAllLifecyclePhases()
#39 0x7f7b46ca1e4b blink::PageAnimator::UpdateAllLifecyclePhases()
#40 0x7f7b46ca4175 blink::PageWidgetDelegate::UpdateAllLifecyclePhases()
#41 0x7f7b42389cb7 blink::WebViewImpl::UpdateAllLifecyclePhases()
#42 0x7f7b46399b52 blink::WebViewFrameWidget::UpdateAllLifecyclePhases()
#43 0x7f7b3dec498b content::RenderWidget::UpdateVisualState()
#44 0x7f7b3dcc866a content::RenderWidgetCompositor::UpdateLayerTreeHost()
#45 0x7f7b49ad70ed cc::LayerTreeHost::RequestMainFrameUpdate()
#46 0x7f7b49bc79ac cc::ProxyMain::BeginMainFrame()
#47 0x7f7b49bc48b4 <unknown>
#48 0x7f7b49bc475f <unknown>
#49 0x7f7b49bc46c8 <unknown>
#50 0x7f7b49bc45de <unknown>
#51 0x7f7b3a97b09e <unknown>
#52 0x7f7b3a9c5a8e base::debug::TaskAnnotator::RunTask()
#53 0x7f7b43361674 blink::scheduler::TaskQueueManager::ProcessTaskFromWorkQueue()
#54 0x7f7b4335e766 blink::scheduler::TaskQueueManager::DoWork()
#55 0x7f7b43369b9c <unknown>
#56 0x7f7b43369acf <unknown>
#57 0x7f7b43369a43 <unknown>
#58 0x7f7b4336995c <unknown>
#59 0x7f7b3a97b09e <unknown>
#60 0x7f7b3a9c5a8e base::debug::TaskAnnotator::RunTask()
#61 0x7f7b3aa5b6cd base::MessageLoop::RunTask()
  r8: 0000000000000000  r9: 00007ffcfb5e9050 r10: 0000000000000008 r11: 0000000000000246
 r12: 0000000000cad9cb r13: 00007ffcfb5f8140 r14: 0000000000000000 r15: 0000000000000000
  di: 0000000000000002  si: 00007ffcfb5e9050  bp: 00007ffcfb5e92a0  bx: 0000000000000000
  dx: 0000000000000000  ax: 0000000000000000  cx: 00007f7b2ea880bb  sp: 00007ffcfb5e9050
  ip: 00007f7b2ea880bb efl: 0000000000000246 cgf: 002b000000000033 erf: 0000000000000000
 trp: 0000000000000000 msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

**Comment by ckerr** [comment]
That error is from `libchromiumcontent/third_party/WebKit/Source/core/animation/ElementAnimations.cpp`:

```c++
void ElementAnimations::UpdateBaseComputedStyle(
    const ComputedStyle* computed_style) {
  if (!IsAnimationStyleChange()) {
    base_computed_style_ = nullptr;
    return;
  }
#if DCHECK_IS_ON()
  if (base_computed_style_ && computed_style)
    DCHECK(*base_computed_style_ == *computed_style);
#endif
  base_computed_style_ = ComputedStyle::Clone(*computed_style);
}
```

**Comment by zcbenz** [comment]
Looks like https://bugs.chromium.org/p/chromium/issues/detail?id=701250

**Comment by alexeykuzmin** [comment]
> https://bugs.chromium.org/p/chromium/issues/detail?id=701250

The [latest change](https://codereview.chromium.org/2757543002) related to this issue landed in 59.0.3062.0.

**Comment by miniak** [comment]
Seems to be fixed now.
